src/call-session/outbound.ts

```diff
@@ -8,14 +8,15 @@ import {
   fakeDomain,
   fakeEmail,
   generateAuthorization,
-  uuid,
   withoutTag,
 } from "../utils.js";
 
 class OutboundCallSession extends CallSession {
-  public constructor(webPhone: WebPhone) {
+  public constructor(webPhone: WebPhone, callee: string) {
     super(webPhone);
     this.direction = "outbound";
+    this.localPeer = `<sip:${webPhone.sipInfo.username}@${webPhone.sipInfo.domain}>;tag=${this.id}`;
+    this.remotePeer = `<sip:${callee}@${webPhone.sipInfo.domain}>`;
```

Comment:
It seems that you need to access remotePeer(callee number) before the `call(...)`?